app/pb_kits/playbook/pb_home_address_city/_home_address_city.html.erb

```diff
@@ -0,0 +1,8 @@
+<%= content_tag(:div,
```

Comment:
Did you consider using an [address tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address) for this? It would provide a more semantic way of presenting this information.